Github workflow not working anymore

After the last upgrade, the workflow fails and can't find the arduino board anymore. Have to figure out what's happening there.

```
Run mkdir -p build
/home/runner/arduino-1.8.19
-- Found Arduino Platform: /home/runner/arduino-1.8.19/hardware/arduino/avr
-- Selected Arduino Board:  []
CMake Error at arduino-toolchain/Platform/Arduino-Determine.cmake:3 (message):
  Please select a valid arduino board and its menu options using one of the
  below methods.
  1.  From CMake GUI
  2.  From the generated BoardOptions.cmake at
  /home/runner/work/nuki_hub/nuki_hub/build/BoardOptions.cmake
  3.  Use yor own preset BoardOptions.cmake
  -DARDUINO_BOARD_OPTIONS_FILE=<file>
  4.  Use -DARDUINO_BOARD=<board_id> and
  -DARDUINO_<BOARD_ID>_MENU_<MENU_ID>_<MENU_OPT_ID>=<TRUE/FALSE>!!!
Call Stack (most recent call first):
  /usr/local/share/cmake-3.23/Modules/CMakeDetermineSystem.cmake:160 (include)
  CMakeLists.txt:5 (project)
CMake Error: CMake was unable to find a build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -18,14 +18,8 @@ jobs:
           tar -xf /tmp/arduino-ide.tar.xz --directory ~/
           cd ~/arduino*
           ./install.sh
-      - name: Install Arduino CLI
-        uses: arduino/setup-arduino-cli@v1
-      - name: Install ESP32 SDK
-        run: |
-          arduino-cli config init
-          arduino-cli config add board_manager.additional_urls https://raw.githubusercontent.com/technyon/nuki_hub/master/.github/dist/package_esp32_index_2_0_4.json
-          arduino-cli core update-index
-          arduino-cli core install esp32:esp32
+          ./arduino --pref "boardsmanager.additional.urls=https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_index.json" --save-prefs
+          ./arduino --install-boards esp32:esp32
       - name: Install Arduino CMake Toolchain
         uses: actions/checkout@v2
         with:
@@ -50,7 +44,7 @@ jobs:
           cp build/nuki_hub.bin release/
           cp build/nuki_hub.partitions.bin release/
           cp $(find ~/.arduino15/packages/esp32/ | grep boot_app0.bin) release/
-          cp $(find ~/.arduino15/packages/esp32/ | grep bootloader_dio_80m.bin) release/
+          cp $(find ~/.arduino15/packages/esp32/ | grep esp32/bin/bootloader_dio_80m.bin) release/
           echo "esptool.py --chip esp32 --port /dev/ttyUSB0 --baud 921600 --before default_reset --after hard_reset write_flash -z --flash_mode dio --flash_freq 80m --flash_size detect 0xe000 boot_app0.bin 0x1000 bootloader_dio_80m.bin 0x10000 nuki_hub.bin 0x8000 nuki_hub.partitions.bin" > release/flash.sh
       - uses: actions/upload-artifact@v3
         with:

